snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/rf2/SnomedRf2ExportRequest.java

```diff
@@ -216,8 +218,73 @@ void setExtensionOnly(final boolean extensionOnly) {
 		this.extensionOnly = extensionOnly;
 	}
 	
+	public void setCodeSytemId(String codeSytemId) {
+		this.codeSystemId = codeSytemId;
+	}
+
+	private CodeSystem getCodeSystem(String codeSystemId, RepositoryContext context) {
+
+		return CodeSystemRequests.prepareGetCodeSystem(codeSystemId)
+				.setExpand("settings()")
+				.buildAsync()
+				.execute(context.service(IEventBus.class))
+				.getSync();
+
+	}
+	
+	private String getCountryNamespaceElement(Rf2MaintainerType maintainerType, String nrcCountryCode) {
+		final StringBuilder builder = new StringBuilder();
+		
+		switch (maintainerType) {
+			case NRC:
+				builder.append(nrcCountryCode);
+				break;
+			case OTHER_EXTENSION_PROVIDER:
+				// Nothing to append
+				break;
+			case SNOMED_INTERNATIONAL:
+				builder.append("INT");
+				break;
+			default:
+				break;
+		}
+		
+		return builder.toString();
+	}
+	
 	@Override
 	public Attachment execute(final BranchContext context) {
+		
+		SnomedExportDefaultConfiguration exportDefaultConfiguration = ApplicationContext.getServiceForClass(SnomedCoreConfiguration.class).getExport();
```

Comment:
Please remove the `snomed.export.*` configuration keys, you are replacing these with the ones in the CodeSystem settings.